micrometer-commons/src/main/java/io/micrometer/common/Tag.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.micrometer.observation;
+package io.micrometer.common;
```

Comment:
`Tag` and `Tags` are tricky because those can be used by third-party instrumentations.